content/en/service_management/service_level_objectives/guide/slo_types_comparison.md

```diff
@@ -0,0 +1,54 @@
+---
+title: SLO Type Comparison
+kind: Guide
+further_reading:
+- link: "/service_management/service_level_objectives/"
+  tag: "Documentation"
+  text: "Overview of Service Level Objectives"
+- link: "/service_management/service_level_objectives/metric/"
+  tag: "Documentation"
+  text: "Metric-based SLOs"
+- link: "/service_management/service_level_objectives/monitor/"
+  tag: "Documentation"
+  text: "Monitor-based SLOs"
+---
+
+## Overview
+
+When creating SLOs, you can choose from the following types:
+
+**Metric-based SLOs**: can be used for count-based data streams, the SLI is based on the sum of good events divided by the sum of total events.
```

Comment:
I recommend using one bulleted list here, with a bolded entry for each type. You'll have to repeat the one sentence that applies to two of the types, but the structure will be easier to read.